packages/office-ui-fabric-react/src/components/ColorPicker/ColorSlider/ColorSlider.types.ts

```diff
@@ -0,0 +1,47 @@
+import { ITheme, IStyle } from '../../../Styling';
+import { IStyleFunctionOrObject } from '../../../Utilities';
+
+export interface IColorSlider {
```

Comment:
Can remove this?

Reply:
It's used in the `componentRef` property of `IColorSliderProps`. This follows the pattern of other components (e.g. `Callout`, `ChoiceGroup`, `Beak`, `Coachmark`, etc.) that expose empty interfaces to use in their `componentRef` property.

Reply:
I understand the usage of componentRef. My question revolves around this specific interface. Why can't we just use `componentRef?: () => void;` below and get rid of this interface? This interface is empty and provides no type value. There are other empty interfaces similar to this too throughout this PR.

Reply:
I see that it's empty elsewhere but I'm still not clear on the reasoning...